frontend/src/components/projects/ProjectSearch.tsx

```diff
@@ -16,7 +16,7 @@ const ProjectSearch: FunctionComponent = () => {
               async (response) => {
                 if (response.status === 200) {
                   const data = await response.json();
-
+                  console.log(data);
```

Comment:
nit: I think we would want to remove console logging for info or debug purposes in production and leave production console logging for warning or errors (to reduce noise)

Reply:
Oh sorry, missed that, I'll remove it